packages/waku/src/lib/builder/build.ts

```diff
@@ -364,6 +371,7 @@ const buildClientBundle = async (
       rscManagedPlugin(config),
     ],
     build: {
+      emptyOutDir: !partial,
```

Comment:
I wonder if we can make this always `true` and empty `dist` folder at the beginning of `build()` function.

Reply:
It would make the diff simpler. I just did not want to potentially affect the existing behavior. If you feel comfortable trying that, I have no objections.

Reply:
Yeah, I would have said so (=avoid potentially affect the current behavior).

I just checked the vite code around emptyOutDir, and it doesn't seem it's very straightfoward as I thought.

So, let's use `emptyOutDir: !partial` this time.